cpp/autosar/src/qlpack.yml

```diff
@@ -1,4 +1,6 @@
-name: autosar-cpp-coding-standards
+name: codeql/autosar-cpp-coding-standards
 version: 2.13.0-dev
 suites: codeql-suites
-libraryPathDependencies: common-cpp-coding-standards
+dependencies:
```

Comment:
We should add the `defaultSuiteFile`, `suites`, `license`, and `description` properties.

The `defaultSuiteFile` helps with running the default selected suite when our package is provided to `codeql database analyze` and the `suites` informs the CodeQL cli of the possible suites and turns them into well-known suites so you can run them without providing the full path.